src/glide/browser/base/content/browser.mts

```diff
@@ -1889,7 +1897,20 @@ function make_glide_api(): typeof glide {
           debug: Services.prefs.getBoolPref("devtools.testing", false),
         });
       },
+
+      label_prefix_free(hints): string[] {
```

Comment:
note: you'll need to define this function, and the one below, in `glide.d.ts` as well

Reply:
Thanks, done. And added docstrings so that they show up in the API documentation.